I/iso_codes/build_tarballs.jl

```diff
@@ -34,7 +36,7 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [AnyPlatform()]
+platforms = supported_platforms(; experimental=true)
```

Comment:
`experimental` does nothing, it's no-op.

@staticfloat would
```julia
[AnyPlatform(); supported_platforms(; exclude=!Sys.iswindows)]
```
do the right thing, instead of having 14 identical artifacts?

Reply:
Sadly, if you have a platformless binding in an `Artifacts.toml` file, you can't also have other platform-specific bindings in that same `Artifacts.toml` file.  So the best we can do here is to just have the same tarball listed for a bunch of different platforms, then a different tarball for the windows platforms.